drivers/clock_control/Kconfig.stm32

```diff
@@ -80,15 +81,14 @@ config CLOCK_STM32_MCO1_SRC_HSE
 	           SOC_SERIES_STM32F4X || \
 	           SOC_SERIES_STM32F7X || \
 	           SOC_SERIES_STM32L4X || \
-	           SOC_SERIES_STM32H7X || \
-	           SOC_SERIES_STM32H7RSX || \
-	           SOC_SERIES_STM32H5X
+		   SOC_SERIES_STM32U5X
```

Comment:
Is removal of H5 / H7 / H7R/S series a mistake?

Reply:
Must be a mistake during rebase, fixed.